src/content/en/tools/chrome-devtools/_toc.yaml

```diff
@@ -53,8 +53,8 @@ toc:
       section:
       - title: Get Started
         path: /web/tools/chrome-devtools/javascript/
-      - title: Set Breakpoints
-        path: /web/tools/chrome-devtools/javascript/add-breakpoints
+      - title: Breakpoints Guide
```

Comment:
Is there any particular reason you're moving away from a title beginning with a verb? After reading the article, I think 'Set Breakpoints' is more appropriate.